### PR TITLE
Using output environment file

### PR DESCRIPTION
**Pull request by mgiamberardino**

### CODE REVIEW

File: schema-conformance/action.yaml [hunk 1]
```diff
@@ -89,9 +89,11 @@ runs:
     - id: map-outputs
       if: always()
       run: |
-        echo "::set-output name=success::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_succ }}"
-        echo "::set-output name=failed::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_fail }}"
-        echo "::set-output name=error::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_error }}"
-        echo "::set-output name=skip::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_skip }}"
-        echo "::set-output name=total::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests }}"
+        echo "${{ steps.publish-junit.outputs.json }}"
+        cat $PWD/reports/junit.xml
```
Comment (buchi-busireddy) [inline schema-conformance/action.yaml:93]: Are you planning to remove this later? Maybe we can move this under `debug: true` condition?

File: schema-conformance/action.yaml [hunk 2]
```diff
@@ -89,9 +89,11 @@ runs:
...
+        echo "success=${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_succ }}" >> $GITHUB_OUTPUT
```
Comment (buchi-busireddy) [inline schema-conformance/action.yaml:94]: How about writing all these to a file at once and piping that file output to GH_OUTPUT? we can also cat that file in debug mode and easily be able to debug if there are any syntax issues or something.

**Review by buchi-busireddy — APPROVED** [review]

**Comment by buchi-busireddy** [comment]
@mgiamberardino why is the success count `0` in the schema-conformance test output? I'd expect it to be `9` because there are 3 endpoints and in total we have 5 checks/tests for each endpoint but 3 checks/tests are succeeding across all endpoints so it should be 9 successful tests.

**Comment by mgiamberardino** [comment]
> @mgiamberardino why is the success count `0` in the schema-conformance test output? I'd expect it to be `9` because there are 3 endpoints and in total we have 5 checks/tests for each endpoint but 3 checks/tests are succeeding across all endpoints so it should be 9 successful tests.

@buchi-busireddy This is not how JUnit report is working. JUnit reporter is considering each endpoint/generation method pair a test case and each check a failure withing that check. And even now there is a bug where successful checks/assertions are printed as failures within a test case so every test case is considered as a failure right now.

**Pull request merged**